src/components/views/toasts/VerificationRequestToast.tsx

```diff
@@ -38,6 +38,7 @@ interface IProps {
 interface IState {
     counter: number;
     device?: DeviceInfo;
+    IP?: string;
```

Comment:
I'd suggest spelling this `ip`, since we use e.g. `uri` for URIs and other similar acronyms.